graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -186,36 +230,35 @@ class RenameSchemaTypesVisitor(Visitor):
         }
```

Comment:
Do we specifically need this to be a class attribute here? If not, can we move it to module-level and make it sit right adjacent to the `Union` definition that it duplicates?

Reply:
It's specifically used for `RenameSchemaTypesVisitor`, as it explicitly leaves out renaming the schema query type (i.e. `SchemaQuery`) and has `RenameQueryTypeFieldsVisitor` do that instead. Moving it to module-level could be confusing considering that we do actually modify the schema query type, just not in this particular class.